includes/wp-admin/class-settings.php

```diff
@@ -186,6 +186,17 @@ public static function register_settings() {
 			)
 		);
 
+		\register_setting(
+			'activitypub_advanced',
+			'activitypub_ap_post_purge_days',
+			array(
+				'type'         => 'integer',
+				'description'  => \__( 'Number of days to keep remote posts.', 'activitypub' ),
+				'default'      => 180,
+				'show_in_rest' => false,
```

Comment:
```suggestion
				'description'  => 'Number of days to keep remote posts.',
				'default'      => 30,
```
I'd keep it much lower, like a week or a month. Is `show_in_rest` not false by default?